src/manipulate/Events.js

```diff
@@ -24,20 +24,20 @@ const _ontologyIdsForRowIndex = (heatmapData, y) => (
     .filter(onlyUnique)
 )
 
-const onClickUseGenomeBrowser = ({heatmapData, currentGenomeBrowser,heatmapConfig: {experiment, atlasUrl, outProxy}}) => (
-  currentGenomeBrowser && experiment && currentGenomeBrowser !== `none`
-    ? (x, y) => {
+const onClickUseGenomeBrowser = ({heatmapData, heatmapConfig: {experiment, atlasUrl, outProxy}}) => (
+  experiment ?
+    (x, y, genomeBrowser) => {
       window.open(outProxy + URI(experiment.urls.genome_browsers, atlasUrl).addSearch({
         experimentAccession: experiment.accession,
-        name: currentGenomeBrowser,
+        name: genomeBrowser,
```

Comment:
Is there any reason to rename `currentGenomeBrowser` as `genomeBrowser`?

Reply:
The field `name` could update successfully, but I can not load the link successfully for `ensemble` browser option. Is the URL builder correct or something wrong with a particular experiment?

Reply:
Addition point here, do we check the link reachability before offering a browser option?

Reply:
> Is there any reason to rename `currentGenomeBrowser` as `genomeBrowser`?
It felt more intuitive to me to use `genomeBrowser` as a parameter name rather than `currentGenomeBrowser`. Nothing other than this. I can revert if `currentGenomeBrowser` look better


> The field `name` could update successfully, but I can not load the link successfully for `ensemble` browser option. Is the URL builder correct or something wrong with a particular experiment?
Did the url gave error? I didn't understand the issue you are facing. 

> Addition point here, do we check the link reachability before offering a browser option?
No, at the moment we do not. But not sure if we want to. @alfonsomunozpomer can you please guide on this issue.

Reply:
I have got a `500`for `Ensemble` browser.

Reply:
Weird. I'm not getting that!